tests/test_mask.py

```diff
@@ -35,26 +44,70 @@ def assertMask(self, mask: Mask):
             assert 0 <= property < 1, f"{propertyName} is out of range [0,1]"
 
     @staticmethod
-    def assertMaskReply(maskDict: dict):
+    def assertMaskPropertyReply(replyMask: Mask, expectedProperty: str, lowerThreshold: float = 0.9):
         """
-        Validate mask reply
+        Function check estimate property from reply
+
         Args:
-            maskDict: mask estimation result
+            replyMask: mask estimation object
+            expectedProperty: expected property of the mask object
+            lowerThreshold: lower threshold of estimation
         """
-        assert (
-            jsonValidator(schema=MASK_SCHEMA).validate(maskDict) is None
-        ), f"{maskDict} does not match with schema {MASK_SCHEMA}"
+        lowerProbabilitySet = set(MASK_PROPERTIES) - {expectedProperty}
+        expectedResult = getattr(replyMask, expectedProperty)
+        assert expectedResult > lowerThreshold, f"Wrong value for best estimate: '{expectedProperty}'"
```

Comment:
may be `Expected value of the Mask estimation '{}' is less than {lowerThreshold}`, similar to message below?

Reply:
yep